jscomp/others/js_global.ml

```diff
@@ -46,9 +47,9 @@ let procrastinate mins =
   Js.Global.setTimeout work (mins * 60 * 1000)
 ]}
 *)
-external clearTimeout : timeoutId -> unit = "" [@@bs.val]
 
 
+external setInterval : (unit -> unit) -> int -> intervalId = "" [@@bs.val]
 (** {i repeatedly} executes a callback with a specified interval (in milliseconds) between calls
```

Comment:
for safety, can you add `(unit -> unit [@bs.uncurry)`

Reply:
@bobzhang Done.